.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If you want *faster* PR reviews, read how: https://github.com/helm/helm/blob/master/CONTRIBUTING.md
+-->
+
+**What this PR does / why we need it**:
+
+**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+
+**Special notes for your reviewer**:
+
+**If applicable**:
+- [ ] documentation
+- [ ] unit tests
+- [ ] tested backward compatibility (i.e. deploy with previous version, upgrade with this branch)
+
+**Release note**:
```

Comment:
If we start using a release note section we should do two things:

1. Add a label to the issue (automatically?) noting the release notes
1. Make sure the issue is filed in a proper milestone

This way someone performing a release can quickly get a list of notes to include.

Reply:
Yeah, we would need to get the automation in place for that

Reply:
Yeah, let's remove this for now until someone writes up automation for this.